spock-testkit/testkit.gradle

```diff
@@ -55,3 +55,29 @@ task consoleLauncherTest(type: JavaExec) {
   args("--reports-dir", reportsDir)
 //  systemProperty("java.util.logging.manager", "org.apache.logging.log4j.jul.LogManager")
 }
+
+task errorTestCaseTest(type: JavaExec) {
+  dependsOn(testClasses)
+  def reportsDir = file("$buildDir/test-results")
+  outputs.dir(reportsDir)
+  classpath = sourceSets["test"].runtimeClasspath
+  main = "org.junit.platform.console.ConsoleLauncher"
+  args("--select-class", "spock.testkit.testsources.ErrorTestCase")
+  args("--details", "tree")
+  args("--details-theme", "ascii")
+  args("--include-classname", ".*TestCase")
+  args("--reports-dir", reportsDir)
+  ignoreExitValue = true
+  def out = new ByteArrayOutputStream()
+  errorOutput = out
+  standardOutput = out
+
+  doLast {
+    def output = out.toString()
+    println output
+    if (!output.contains("No such property: b for class")) {
```

Comment:
@marcphilipp is there really not a better way?

Reply:
I'm not sure why this is not a regular test that uses JUnit's Test Kit. 🤔

Reply:
Because auf this: https://github.com/spockframework/spock/pull/1102#discussion_r385144384
Or did I miss something?

Reply:
Right, I forgot about that.

You could use the Launcher API instead:

```groovy
def launcher = LauncherFactory.create(LauncherConfig.builder()
	.enableTestEngineAutoRegistration(false)
	.enableTestExecutionListenerAutoRegistration(false)
	.addTestEngines(new SpockEngine())
	.build())
def testPlan = launcher.discover(LauncherDiscoveryRequestBuilder.request()
	.selectClass(ErrorTestCase)
	.build())
assert testPlan.getChildren(testPlan.roots.first()).size == 1
```

Reply:
Ah, perfect, thanks, that's much nicer.